### ISSUE TITLE
 Error File to Prompt for ChatGPT and Bard 1.1.0 

### ISSUE THREAD

**Issue by DAVIDMCCREADY** [fernandodilland/file-to-prompt#4]
One Chrome Version 114.0.5735.199 (Official Build) (64-bit) the following error occurs when trying to use Bard.
Uncaught TypeError: Cannot read properties of undefined (reading 'onClicked')
Context
https://bard.google.com/
Stack Trace
static/background.js:1 (anonymous function)
1
2
3
4
5
6
7
8
chrome.action.onClicked.addListener((tab) => {
    if (tab.id) {
      chrome.scripting.executeScript({
        target: { tabId: tab.id },
        files: ['content.js'],
      });
    }
  });

**Comment by fernandodilland** [comment]
That's right @DAVIDMCCREADY , I'm working on trying to make it work in Bard and Bing, if you have experience I invite you to make a branch and make a pull request with your improvement
